### PR TITLE
Added "get adr algorithms" endpoint 

### PR DESCRIPTION
**Pull request by MStenkaer**
Added endpoint for retrieving the adr algorithms for the default network server, which uses the endpoint `/api/network-servers/{network_server_id}/adr-algorithms` from the ChirpStack network server api.

Needed for resolving https://github.com/OS2iot/OS2IoT-frontend/issues/103

### CODE REVIEW

**Comment by ramogens** [comment]
Thank you very much for making this PR, @MStenkaer. We have started the process of getting a review, and as of now we'll be able to do it in October or November.

File: src/services/chirpstack/network-server.service.ts [hunk 1]
```diff
@@ -68,4 +69,9 @@ export class ChirpstackSetupNetworkServerService
 
         return createNetworkServerDto;
     }
+
+    public async getAdrAlgorithmsForDefaultNetworkServer(): Promise<ListAllAdrAlgorithmsResponseDto> {
+        const networkServerId: string = await this.getDefaultNetworkServerId();
```
Comment (AramAlsabti) [inline src/services/chirpstack/network-server.service.ts:74]: Strictly not necessary to specify type as the method return type is enforced

**Review by AramAlsabti — CHANGES_REQUESTED** [review]
Nicely done. I only have a minor suggestion

File: src/services/chirpstack/network-server.service.ts (hunk 1 above)
Comment (MStenkaer) [reply src/services/chirpstack/network-server.service.ts]: Thank you for the comment. I am not completely sure if it is just the explicit type declaration for the string you

Comment (AramAlsabti) [reply src/services/chirpstack/network-server.service.ts]: That's exactly it.

**Review by AramAlsabti — APPROVED** [review]

**Pull request merged**